getting-started-finding-data/finding-data.md

```diff
@@ -7,20 +7,96 @@ url: https://data.norge.no/getting-started/finding-data
+
 ## Finner du ikke det du leter etter?
 
-### Etterspørsler
+Vet du ikke om data du ser etter finnes? Er det data du trenger som ikke er delt enda? Hvis datasettet ikke er beskrevet i felles datakatalog eller på tilbyders nettsider, bør du gå i dialog med de som tilbyr dem.
 
-### Datajegeren
+### Datalandsbyen: Etterspørre datasett og API-er
 
-### Hvem har ansvar for hva på data.norge.no
+Datalandsbyen er et nettforum dedikert til dialog mellom tilbydere og konsumenter av data. Her kan du etterlyse datasett og API-er du ønsker tilgang til eller har spørsmål om. Du kan også be om utvidelser av eksisterende datasett og API-er, for eksempel mer tilgjengelig data eller endringer i API-funksjonalitet. Besøk brukerforumet [Datalandsbyen](https://datalandsbyen.norge.no/category/6/ettersp%C3%B8r-datasett-og-api-er) for å legge inn din etterspørsel. En veiledning om hvordan du går frem finner du [her](https://datalandsbyen.norge.no/topic/56/ettersp%C3%B8rr-data-og-api).
+
+FDK tilbyr en oversikt over alle [publiserte etterspørsler i Datalandsbyen](https://data.norge.no/requests).
+
+#### Om datalandsbyen
+
+Digdir, Datafabrikken og Statens vegvesen har lansert nettforumet [Datalandsbyen](https://datalandsbyen.norge.no) for å fremme åpen og datadreven innovasjon gjennom samarbeid, dialog og kunnskapsdeling. Dette forumet fungerer som en felles møteplass for alle interesserte i å lære mer om dataforvaltning og ønsker å engasjere seg med både datakonsumenter og datatilbydere.
+
+I Datalandsbyen kan ulike brukergrupper inspirere hverandre med nyttige tips og spørsmål, diskutere bruken av data og API-er, samt etterlyse spesifikke datasett de ønsker tilgang til. Forumet gir også mulighet til å vise eksempler på hvordan data benyttes i tjenester og applikasjoner, samt å dele relevante arrangementer.
+
+Datalandsbyen er åpen for alle, men for å delta aktivt må man registrere seg. Målet er å fremme verdiskapende bruk av data for samfunnet gjennom økt kunnskap, åpenhet og innovasjon.
+
+### Datajegeren: Finn og få tilgang til offentlige datasett
+
+Datajegeren er en tjeneste som hjelper deg med å finne og få tilgang til datasett fra offentlig sektor. Selv om mange datasett er tilgjengelige via Felles Datakatalog, er den ikke komplett. Datajegeren trer inn for å hjelpe bedrifter med å finne og få tilgang til spesifikke datasett.
+
+Ved å besøke tjenesten får du tilgang til et enkelt [kontaktskjema](https://data.norge.no/forms/nb/data-hunter). Her kan du beskrive hva du leter etter, med noen felt som er obligatoriske og noen for å spesifisere forespørselen din. Når skjemaet sendes, går det direkte til ansatte i Digitaliseringsdirektoratet som undersøker hvilken etat som eier dataene og tar seg av den nødvendige kontakten for å skaffe dem.
+
+## Hvem har ansvar for hva på data.norge.no
 
 [Skrive noe om hvilken type informasjon virksomhetene er ansvarlig for, og hvilken informasjon Digdir ved data.norge.no er ansvarlig for].
```

Comment:
### Datatilbyderne

Virksomhetene som eier dataene som er beskrevet på data.norge.no har ansvar for selve innholdet i beskrivelsene og for at dataene du får tilgang til stemmer overens med beskrivelsene. 

Hvis du finner noe feil med beskrivelsene eller tilgangen til dataene, kan du kontakte virksomheten som eier beskrivelsen direkte. Kontaktinformasjon vil som regel være knyttet til beskrivelsen. Hvis du er logget inn i Datalandsbyen, kan du også legge igjen en kommentar på selve beskrivelsen på data.norge.no. Denne kommentaren vil vises både på data.norge.no og i Datalandsbyen. Slike kommentarer og påfølgende diskusjon kan være nyttig for andre brukere som trenger de samme dataene som deg. 

### Digitaliseringsdirektoratet

Digitaliseringsdirektoratet har ansvar for informasjon om når beskrivelser ble publisert på data.norge.no, metadatakvalitet, Datajegeren, KI-oversikt, informasjonssider og at de tekniske løsningene på data.norge.no og Datalandsbyen fungerer som de skal. Digitaliseringsdirektoratet følger også opp kommentarer i Datalandsbyen og purrer på virksomheter som ikke svarer opp kommentarer knyttet til beskrivelser i katalogen.

Reply:
Lagt til 👆